examples/nextjs-demo/src/styles/globals.css

```
@@ -1,33 +1,70 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
-
-:root {
-  --foreground-rgb: 0, 0, 0;
-  --background-start-rgb: 214, 219, 220;
-  --background-end-rgb: 255, 255, 255;
+@import 'tailwindcss';
+@import 'tailwindcss/utilities';
+@import "../../../../packages/ui/src/tailwind.css" layer(theme);
+@plugin "tailwindcss-animate";
+
+@source "../../../packages/ui";
```

```suggestion
@source "../../../../packages/ui";
```
Fixes the nextjs example styles